cabby/geo/util.py

```diff
@@ -486,6 +486,33 @@ def get_line_length(line: LineString) -> float:
   return dist
 
 
+def point_from_list_coord(coord: Sequence) -> Point:
+  '''Converts coordinates in list format (latitude and longtitude) to Point. 
+  E.g, of list [40.715865, -74.037258].
+  Arguments:
+    coord: A lat-lng coordinate to be converted to a point.
+  Returns:
+    A point.
+  '''
+  lat = coord[0]
+  lon = coord[1]
+
+  return Point(lon, lat)
+
+def coords_from_list_coord(coord: Sequence) -> CoordsYX:
+  '''Converts coordinates in list format (latitude and longtitude) to 
+  coordinates in a tuple format. 
+  E.g, of string [40.715865, -74.037258].
+  Arguments:
+    coord: A lat-lng coordinate to be converted to a tuple.
+  Returns:
+    A tuple of lat-lng.
+  '''
+  lat = coord[0]
+  lon = coord[1]
+
+  return CoordsYX(lat, lon)
```

Comment:
It would be preferable to limit the number of functions needed to do these basic transformations and compose them to get what we need. E.g. here we could do  `util.tuple_from_point(point_from_list_coord(coord))` rather than adding the `coords_from_list_coord` function.